### PR TITLE
Delete duplicate ioctl commands related to contract

### PR DESCRIPTION
**Pull request by RuneAkko**
#2188

### CODE REVIEW

File: ioctl/cmd/action/action.go [hunk 1]
```diff
@@ -91,8 +91,6 @@ func (m *sendMessage) String() string {
 func init() {
 	ActionCmd.AddCommand(actionHashCmd)
 	ActionCmd.AddCommand(actionTransferCmd)
-	ActionCmd.AddCommand(actionDeployCmd)
-	ActionCmd.AddCommand(actionInvokeCmd)
```
Comment (dustinxie) [inline ioctl/cmd/action/action.go:95]: have we tested contractDeploy/Invoke command to deploy contract and invoke calls? and new contract command is equivalent with these 2?

Comment (RuneAkko) [reply ioctl/cmd/action/action.go:95]: > have we tested contractDeploy/Invoke command to deploy contract and invoke calls? and new contract command is equivalent with these 2?

we have tests for contract Deploy/Invoke command and examples can be found in [https://github.com/iotexproject/iotex-docs/blob/master/docs/ioctl.md](url). And issue #2188 claims that these are equivalent with action commands.

File: ioctl/cmd/action/actiondeploy.go [hunk 2]
```diff
@@ -1,63 +0,0 @@
...
-
-// actionDeployCmd represents the action deploy command
-var actionDeployCmd = &cobra.Command{
-	Use:   config.TranslateInLang(deployCmdUses, config.UILanguage),
-	Short: config.TranslateInLang(deployCmdShorts, config.UILanguage),
-	Args:  cobra.MaximumNArgs(1),
-	RunE: func(cmd *cobra.Command, args []string) error {
-		cmd.SilenceUsage = true
-		err := deploy(args)
-		return output.PrintError(err)
```
Comment (dustinxie) [inline ioctl/cmd/action/actiondeploy.go:40]: can we return a message
"ioctl action deploy has been deprecated, use ioctl contract deploy instead"

all code below like init(), deploy() can be deleted

File: ioctl/cmd/action/actioninvoke.go [hunk 3]
```diff
@@ -1,67 +0,0 @@
...
-
-// actionInvokeCmd represents the action invoke command
-var actionInvokeCmd = &cobra.Command{
-	Use:   config.TranslateInLang(invokeCmdUses, config.UILanguage),
-	Short: config.TranslateInLang(invokeCmdShorts, config.UILanguage),
-	Args:  cobra.RangeArgs(1, 2),
-	RunE: func(cmd *cobra.Command, args []string) error {
-		cmd.SilenceUsage = true
-		err := invoke(args)
-		return output.PrintError(err)
```
Comment (dustinxie) [inline ioctl/cmd/action/actioninvoke.go:40]: can we return a message
"ioctl action invoke has been deprecated, use ioctl contract invoke instead"

all code below like init(), deploy() can be deleted

**Pull request merged**